app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_f = command.split()[1]
+    new_f = command.split()[2]
+    if new_f == name_f:
+        return
+    with open(name_f, "r") as file_in, open(new_f, "w") as file_out:
```

Comment:
Ensure that the 'cp' input command is correct. The checklist mentions checking the correctness of the 'cp' input command.